docs/design/autoware-architecture/planning/index.md

```diff
@@ -97,45 +97,45 @@ As mentioned in the goal session, this planning module is designed to be extensi
+| Pull Out                                     | Plan path for pull over to start from the road shoulder. <br> <br> Reference implementation is in [Pull Out Module](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_path_planner/docs/behavior_path_planner_start_planner_design/#:~:text=WIP-,Path%20Generation,-%23). <br> Demonstration Video:[Simple Pull Out](https://youtu.be/xOjnPqoHup4), [Backward Pull Out](https://youtu.be/iGieijPcPcQ)                                                                                                                                                                 | - Lanelet map (shoulder lane)                                               | ![pull-out](image/features-pull-out.drawio.svg)                                 |
+| Path Shift                                   | Plan path in lateral direction in response to external instructions. <br> <br> Reference implementation is in [Side Shift Module](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_path_planner/docs/behavior_path_planner_side_shift_design/).                                                                                                                                                                                                                                                                                                                      | - None                                                                      | ![side-shift](image/features-side-shift.drawio.svg)                             |
+| Obstacle Stop                                | Plan velocity to stop for an obstacle on the path. <br> <br> Reference implementation is in [Obstacle Stop Planner](https://autowarefoundation.github.io/autoware.universe/main/planning/obstacle_stop_planner/), [Obstacle Cruise Planner](https://autowarefoundation.github.io/autoware.universe/main/planning/obstacle_cruise_planner/). `launch obstacle_stop_planner` and enable flag: `TODO`, `launch obstacle_cruise_planner` and enable flag: `TODO` <br> Demonstration Videos: [Stop for Obstacle](https://youtu.be/d8IRW_xArcE), [Slow Down for Obstacle](https://youtu.be/gvN1otgeaaw) | - objects information                                                       | ![obstacle-stop](image/features-obstacle-stop.drawio.svg)                       |
```

Comment:
@Tier4Guan I think it is better to move Slow Down for Obstacle to Obstacle Deceleration. What do you think?

Reply:
You are right! Thank you! I will change it!